enhancements/oc/olm-mirroring.md

```diff
@@ -0,0 +1,121 @@
+# OC Tooling for Disconnected OLM
+
```

Comment:
I'm missing a header like this here: https://github.com/openshift/enhancements/blob/master/enhancements/compact-clusters.md look at the source it's the first section in that file.

Reply:
Added